lib/chef/formatters/error_inspectors/compile_error_inspector.rb

```diff
@@ -30,15 +30,16 @@ class CompileErrorInspector
 
         def initialize(path, exception)
           @path, @exception = path, exception
+          @backtrace_lines_in_cookbooks = nil
+          @file_lines = nil
```

Comment:
In the `#file_lines` method. It's technically okay because the code is `@file_lines ||= IO.readlines(culprit_file)` which doesn't trigger a ruby warning for uninitialized variables, but generally I like to create all the instance vars in `initialize` so you can see them all at a glance.